gnomad_qc/v4/create_release/create_combined_faf_release_ht.py

```diff
@@ -353,6 +494,7 @@ def main(args):
             res.check_resource_existence()
             ht = res.freq_ht.ht()
             ht = ht.select(
+                # TODO: this step is very slow, need to optimize
```

Comment:
Maybe include how slow so if you're not the one to pick up the work others have a reference.

Reply:
Also just note that it depends significantly on the `min_cell_count` value